includes/Modules/Idea_Hub.php

```diff
@@ -299,66 +302,7 @@ protected function create_data_request( Data_Request $data ) {
 					);
 				};
 			case 'GET:new-ideas':
-				// @TODO: Implement this with the real API endpoint.
-				return function() {
-					return array(
-						array(
-							'name'   => 'ideas/17450692223393508734',
-							'text'   => 'Why Penguins are guanotelic?',
-							'topics' =>
-								array(
-									array(
-										'mid'          => '/m/05z6w',
-										'display_name' => 'Penguins',
-									),
-								),
-						),
-						array(
-							'name'   => 'ideas/14025103994557865535',
-							'text'   => 'When was sushi Kalam introduced?',
-							'topics' =>
-								array(
-									array(
-										'mid'          => '/m/07030',
-										'display_name' => 'Sushi',
-									),
-								),
-						),
-						array(
-							'name'   => 'ideas/7612031899179595408',
-							'text'   => 'How to speed up your WordPress site',
-							'topics' =>
-								array(
-									array(
-										'mid'          => '/m/09kqc',
-										'display_name' => 'Websites',
-									),
-								),
-						),
-						array(
-							'name'   => 'ideas/2285812891948871921',
-							'text'   => 'Using Site Kit to analyze your success',
-							'topics' =>
-								array(
-									array(
-										'mid'          => '/m/080ag',
-										'display_name' => 'Analytics',
-									),
-								),
-						),
-						array(
-							'name'   => 'ideas/68182298994557866271',
-							'text'   => 'How to make carne asada',
-							'topics' =>
-								array(
-									array(
-										'mid'          => '/m/07fhc',
-										'display_name' => 'Cooking',
-									),
-								),
-						),
-					);
-				};
+				return $this->fetch_ideas( 'new' );
```

Comment:
This datapoint (as well as the one for saved ideas) also need to get a case within the `parse_data_response` method, since only the array of `ideas` within the response object should be returned, not the full response object (similar to how all our other API proxy datapoints work).

In addition to that, we also need to afterwards filter out any ideas from the response for which there already is a WordPress draft or published post - in other words, these endpoints should never return ideas that would also be included in the response from `GET:draft-post-ideas` or `GET:published-post-ideas`.

Reply:
Yes, I know it. I intentionally didn't add it yet. Will do it once I can get ideas on my end.

Reply:
Implemented.